app/presenters/contact_presenter.rb

```diff
@@ -7,35 +7,45 @@ def initialize(contact)
     @contact = contact
   end
 
+  def content_id
+    contact.content_id
+  end
+
+  def update_type
+    "major"
+  end
+
   def present
     {
-      content_id: contact.content_id,
       title: contact.title,
       description: contact.description,
       format: "contact",
       locale: "en",
       publishing_app: "contacts",
       rendering_app: "contacts-frontend",
-      update_type: "major",
```

Comment:
Removal of this might be invalid, see here: https://github.com/alphagov/whitehall/pull/2431#issuecomment-172898454